pkgs/tools/misc/sanctity/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, rustPlatform, fetchFromGitea }:
```

```suggestion
{ lib, rustPlatform, fetchFromGitea }:
```

`stdenv` not used in this derivation